spyder/app/tests/test_mainwindow.py

```diff
@@ -266,6 +266,60 @@ def test_filter_numpy_warning(main_window, qtbot):
     CONF.set('variable_explorer', 'minmax', False)
 
 
+@pytest.mark.slow
+@flaky(max_runs=3)
+@pytest.mark.use_introspection
+@pytest.mark.skipif(PY2,
+                    reason="Times out in PY2")
```

Comment:
Please rewrite these two lines like this:

`@pytest.mark.skipif(PY2, reason="Times out in PY2")`